### ISSUE TITLE
`wd point ..` completes relative to current directory

### ISSUE THREAD

**Issue by ErsafanRend** [mfaerevaag/wd#51]
Thanks for this awesome tool. This is great work.
I'm using `wd` (0.4.6) on `zsh` (5.4.2) with `oh-my-zsh`.
When I have
```
~/wd ❯ tree
.
├── bla
│   └── blub
└── foo
    └── bar
        └── baz
```
and 
```
~/wd ❯ wd
Warp points
baz   -- ~/wd/foo/bar/baz
```
and I want to complete `wd baz ../<TAB>`, I get
```
~/wd❯ cd bla
~/wd/bla❯ wd baz ../
directory
bla/  foo/
```
I would expect `wd baz ../` to point to the parent directory of the warp point and not to the parent directory of the current working directory.

Is this intended behaviour?

**Comment by altschuler** [comment]
Good catch, that is definitely not intended behavior. We'll look into it, thanks! :)

**Comment by alpha-tango-kilo** [comment]
I don't understand how autocompletion scripts worked, I've never worked with them before. By all means have a crack at it if you're interested
